setup_kubernetes.sh

```diff
@@ -25,6 +25,7 @@ svc_cidr=`parse_conf svc_cidr`
 install_deps=`parse_conf install_deps`
 is_master=`parse_conf is_master`
 go_version=`parse_conf go_version`
+is_single_numa=`parse_conf is_single_numa`
```

Comment:
It better to do is topology manager
And allow to configure the 3 supported polices